pkg/blob/blob.go

```diff
@@ -317,7 +317,7 @@ func checkContainerNameBeginAndEnd(containerName string) bool {
 
 // isSASToken checks if the key contains the patterns. Because a SAS Token must have these strings, use them to judge.
 func isSASToken(key string) bool {
-	return strings.Contains(key, "?sv=")
+	return strings.HasPrefix(key, "?")
```

Comment:
add a link to sas token format: https://docs.microsoft.com/en-us/rest/api/eventhub/generate-sas-token
I think it's better set like this: `strings.Contains(key, "&sig=")` per the above sas token format doc

Reply:
The SAS token is actually a query string in URL, the format is like: `?sv={}&ss={}&srt={}&sp={}&se={}&st={}&spr={}&sig={}`. I guess maybe `sig=` could be put right after question mark `?`. Also I think question mark won't appear in access token, so I make the change in this way.

Reply:
Also I will add a link to SAS token format in the comment.

Reply:
the real sas token format actually only contains a few of the above const format, we may only check a few, e.g. `se=`, `sig=`,  and question mark check may be not necessary, as long as the former 2 fixed strings are there, then it's a sas token

Reply:
nvm, let's just use prefix `?` check to make it simple.